semantic-convention-utils/src/main/java/org/hypertrace/semantic/convention/utils/db/DbSemanticConventionUtils.java

```diff
@@ -163,7 +163,33 @@ public static boolean isSqlTypeBackendForOtelFormat(
     if (SpanAttributeUtils.containsAttributeKey(event, SQL_URL)) {
       return Optional.of(SpanAttributeUtils.getStringAttribute(event, SQL_URL));
     }
-    return getBackendURIForOtelFormat(event);
+    Optional<String> backendUrl = getBackendURIForOtelFormat(event);
+    if (backendUrl.isPresent()) {
+      return backendUrl;
+    }
+    if (SpanAttributeUtils.containsAttributeKey(
+        event, OTelDbSemanticConventions.DB_CONNECTION_STRING.getValue())) {
+      String url = SpanAttributeUtils.getStringAttribute(
```

Comment:
Don't we have to convert this `String` to URI object?
```
try {
new URI(URL)
} catch (Mulformed) {
}
```

Reply:
done